yahoo_fantasy_api/yhandler.py

```diff
@@ -370,3 +370,12 @@ def get_game_raw(self, game_code):
         :return: JSON document of the request.
         """
         return self.get("game/{}".format(game_code))
+
+    def get_league_teams_raw(self, league_id):
+            """Return the raw JSON when requesting the teams in a league
+
+            :param league_id: League ID to get the teams for
+            :type league_id: str
+            :return: JSON document of the request.
+            """
+            return self.get("league/{}/teams".format(league_id))
```

Comment:
Alignment looks off here.
```suggestion
    def get_league_teams_raw(self, league_id):
        """Return the raw JSON when requesting the teams in a league

        :param league_id: League ID to get the teams for
        :type league_id: str
        :return: JSON document of the request.
        """
        return self.get("league/{}/teams".format(league_id))
```